packages/explorer/src/app/(explorer)/api/sqlite-indexer/worlds/route.ts

```diff
@@ -1,24 +1,41 @@
-import { Address } from "viem";
-import { getDatabase } from "../../utils/getDatabase";
+import { anvil } from "viem/chains";
+import { indexerForChainId } from "../../../utils/indexerForChainId";
 
 export const dynamic = "force-dynamic";
 
-type Row = {
-  address: Address;
-};
-
-type SqliteTable = Row[] | undefined;
-
 export async function GET() {
   try {
-    const db = getDatabase();
-    const data = (await db?.prepare("SELECT DISTINCT address FROM __mudStoreTables").all()) as SqliteTable;
-    const items = data?.map((row) => ({
+    const indexer = indexerForChainId(anvil.id);
+    const response = await fetch(indexer.url, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify([
+        {
+          query: "SELECT DISTINCT address FROM __mudStoreTables",
+        },
+      ]),
+    });
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! Status: ${response.status}`);
```

Comment:
it would be great if these errors were a little more descriptive, like "Failed to fetch from indexer"

Reply:
also any reason we have to do this fetch in an API route rather than directly from the page that needs to fetch worlds?